apps/chat/src/components/Sidebar/Sidebar.tsx

```diff
@@ -105,7 +105,7 @@ const Sidebar = <T,>({
 
   const resizeTriggerColor = classNames(
     'xl:bg-accent-primary xl:text-accent-primary',
-    isResizing ? 'bg-accent-primary text-accent-primary' : '',
+    isResizing ? 'md:bg-accent-primary md:text-accent-primary' : '',
```

Comment:
```suggestion
    isResizing && 'md:bg-accent-primary md:text-accent-primary',
```
nit

Reply:
This class constant has been removed, according to the following comment.